Dockerfile

```diff
@@ -9,5 +9,7 @@ FROM alpine:latest
 RUN apk --no-cache add ca-certificates
 COPY --from=builder /go/src/github.com/garethr/kubeval/bin/linux/amd64/kubeval .
 RUN ln -s /kubeval /usr/local/bin/kubeval
-ENTRYPOINT ["/kubeval"]
+RUN printf '#!/bin/sh\n\neval kubeval $@' >> /kubeval.sh
+RUN chmod +x /kubeval.sh
```

Comment:
A few ideas:
1) how about combining RUN directives together (to have less layers)?
2) If you create a new file, I'd rather use `>` to make it more obvious your intentions. `>>` is using typically to append something into file
3) `printf` doesn't add a new line to the end, so you have to add it manually. At this moment, the resulted script doesn't end with a new line. While the script will work anyway, it's not accurate :)

Reply:
This PR breaks the support of whitespaces in filenames / would require double-escaping.